fix(search): add level to docs (#6051)
getsentry/sentry-docs: src/docs/product/sentry-basics/search/searchable-properties.mdx

| issue.category | The category of the issue (either `error` or `performance`). | | x | string |
| last_seen() | Datetime when the event was last seen. Equivalent to `max(timestamp)`. Doesn't take a parameter. | x |  | datetime |
| lastSeen | Datetime when the event was last seen. For example, `lastSeen:+30d` returns issues last seen 30 days ago or more; `lastSeen:-2d` returns issues last seen within the past two days. This is similar to `age`. |  | x | datetime |

> | level | Severity of the event (i.e., fatal, error, warning). Always set to info for transactions. | x | x | string |

Reviewer: ```suggestion
| level | Severity of the event (such as: fatal, error, warning). Always set to info for transactions. | x | x | string |
```
